docs/src/catalyst_functionality/dsl_description.md

```diff
@@ -250,7 +250,9 @@ end
 ```
 corresponding to the ODE model
 ```@example tut2
-convert(ODESystem, rn)
+using Latexify
+# latexify(rn; form=:ode) # hide
+latexify(convert(ODESystem,rn))
```

Comment:
Why add the latexify call? The point is that it automatically displays the Latex just fine when converting as the default system display type, and this shows the user how to get the ODEs at the same time...

Reply:
If we just want to display the ODE I think it makes more sense to only use `latexify`, no reason to show get ModelingToolkit stuff involved unless we want to. This was the one that got broken, but once we get it working I think we should use `latexify(rn; form=:ode)` rather than `latexify(convert(ODESystem,rn))`, unless we are in a section explicitly discussing conversion between modelingtoolkit systems.